js/src/elements/form-behavior.ts

```diff
@@ -407,6 +419,13 @@ export class FormBehaviorElement extends HTMLElement {
 				continue;
 			}
 
+			if (input.type === 'radio') {
+				const radioGroup = form.querySelectorAll<HTMLInputElement>(`input[name="${input.name}"]`);
+				if (radioGroup.length > 1 && radioGroup[0] !== input) {
```

Comment:
Can you help me understand why we need to check that the given input isn't the first in its radiogroup?

Reply:
If I do not continue on the first of the group, the report list causes a line items for each radio input.  So instead of just one "radio group" error notification, it would be 1:1 on labels. Not ideal.

This way, when a radio is required, you mark the first with required and all of the same `name` get the applied required logic.

Reply:
This makes sense to me - this might be a useful time for a code comment, something along the lines of:

```ts
// Apply validation to first radio button in group
```